### PR TITLE
Fix(pool page): show deposited liquidity on pool page

### PR DESCRIPTION
**Pull request by ianlapham**
- detect if user has any liquidity deposited into a rewards pool 
- show this along with regular liquidity so users dont get confused about what they own

### CODE REVIEW

File: src/components/PositionCard/index.tsx [hunk 1]
```diff
@@ -157,17 +158,20 @@ export function MinimalPositionCard({ pair, showUnwrapped = false, border }: Pos
...
   const currency1 = unwrappedToken(pair.token1)
 
   const [showMore, setShowMore] = useState(false)
 
-  const userPoolBalance = useTokenBalance(account ?? undefined, pair.liquidityToken)
+  const userDefaultPoolBalance = useTokenBalance(account ?? undefined, pair.liquidityToken)
   const totalPoolTokens = useTotalSupply(pair.liquidityToken)
 
+  // if staked balance balance provided, use instead of fetched balance from pool
+  const userPoolBalance = stakedBalance ?? userDefaultPoolBalance
```
Comment (moodysalem) [inline src/components/PositionCard/index.tsx:173]: shouldn't it be the combination?

File: src/pages/Pool/index.tsx [hunk 2]
```diff
@@ -180,10 +186,19 @@ export default function Pool() {
...
                 </ButtonSecondary>
-
                 {allV2PairsWithLiquidity.map(v2Pair => (
                   <FullPositionCard key={v2Pair.liquidityToken.address} pair={v2Pair} />
                 ))}
+                {stakingPairs.map(
+                  (stakingPair, i) =>
+                    stakingPair[1] && ( // skip pairs that arent loaded
+                      <FullPositionCard
+                        key={stakingPair[1].liquidityToken.address}
```
Comment (moodysalem) [inline src/pages/Pool/index.tsx:196]: conflicting key

File: src/components/PositionCard/index.tsx (hunk 1 above)
Comment (ianlapham) [reply src/components/PositionCard/index.tsx:173]: Guess its a UX question - i could see the case for showing the total, and then giving 3 buttons (add, remove, manage staked)

Comment (ianlapham) [reply src/components/PositionCard/index.tsx:173]: and then a line for the staked amount?

Comment (ianlapham) [reply src/components/PositionCard/index.tsx]: Updated to a combination, with an extra button and row of data if theres a staked amount

**Pull request merged**